Extensions/ArtifactEngine/Engine/cilogger.ts

```diff
@@ -6,15 +6,15 @@ var packagejson = require('../package.json');
 const area: string = 'artifact-engine';
 
 function getDefaultProps() {
+    var hostType = (tl.getVariable('SYSTEM.HOSTTYPE') || "").toLowerCase();
     return {
-        serverurl: tl.getVariable('System.TEAMFOUNDATIONSERVERURI'),
-        releaseurl: tl.getVariable('Release.ReleaseWebUrl'),
-        taskDisplayName: tl.getVariable('Task.DisplayName'),
-        jobid: tl.getVariable('System.Jobid'),
-        releaseid: tl.getVariable('Release.ReleaseId'),
-        builduri: tl.getVariable('Build.BuildUri'),
-        buildid: tl.getVariable('Build.Buildid'),
-        agentVersion: tl.getVariable('Agent.Version'),
+        hostType: hostType,
```

Comment:
serverUrl is not required? it would work for both build & release

Reply:
I need project information also in the url.